File: C Code/Basics/Array/1 D/Power.c
#include<stdio.h>
int main(){
  int a,b;
  printf("Enter base : ");
  scanf("%d",&a);
  printf("Enter Power : ");
  scanf("%d",&b);
  int ans=1;
  for(int i=1;i<=b;i++)
    ans*=a;
  printf("%d raised to power %d is %d .\n",a,b,ans);
  return 0;
}

File: Java Code/Basics/Other/Power.java
import java.util.*;

public class Power{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter the Base  : ");
    float x=sc.nextFloat();
    System.out.print("Enter the Exponent : ");
    int n=sc.nextInt();
    float pow=1;
    if(n>=0){
      for(int i=1;i<=n;i++)
        pow*=x;
    }else{
      for(int i=n;i<0;i++)
        pow/=x;
    }
    System.out.println(x+" raised to power "+n+" = "+pow);
  }
}